dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java

```diff
@@ -95,14 +95,14 @@ public void testRefreshWorkflow() {
         Assert.assertTrue("should find non-zero number of workflows", secondWorkflowCount > 0);
 
         // refresh a specific workflow
-        Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "workflow", "refresh", "--entry", "DockstoreTestUser/dockstore-whalesay-wdl"});
+        Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "workflow", "refresh", "--entry", "github.com/DockstoreTestUser/dockstore-whalesay-wdl"});
```

Comment:
Just in case, I think you can search and replace to use your previously created enum SourceControl.
I doubt we'll change the prefix, but just in case.